src/app/SimpleDownloadDelegate.qml

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2015-2016 Canonical Ltd.
```

Comment:
Files that you're creating should at least have a current copyright date, either for yourself or for the UBports Foundation if you wish to assign your copyright to us.

Reply:
I've updated it to UBports and also to 2021